snippets/parseCookie.md

```diff
@@ -15,12 +15,21 @@ const parseCookie = str =>
     .split(';')
     .map(v => v.split('='))
     .reduce((acc, v) => {
-      acc[decodeURIComponent(v[0].trim())] = decodeURIComponent(v[1].trim());
+      // check if values have double quotes
+      // when yes then slice these double quotes away
+    if(v[1].trim().startsWith("\"") && v[1].trim().endsWith("\"")){
+        acc[decodeURIComponent(v[0].trim())] = decodeURIComponent(v[1].slice(1, v[1].length - 2)); 
```

Comment:
how about using regex? something like this.. 
`v[1].trim().replace(/^"|"$/g, '')`

Reply:
Yes, that's a good solution.
But I have to say I like the way with startsWith and endsWith a bit more.
If you think .replace would make more sense then I can update it :+1:

Reply:
👍